docs/server/project_setup.rst

```diff
@@ -124,15 +124,51 @@ files, in our example :file:`my-project`. Also you must specify the
 **File Types** used in this new project, in our example
 ``Gettext PO (po/pot)``.
 
-Set **Filesystem backend** to ``localfs``. **Path or URL** must point to the
-translation files on the filesystem, in our example
-``{POOTLE_TRANSLATION_DIRECTORY}my-project``. Finally set
-**Path mapping preset** to ``GNU style``.
+The **Filesystem backend** tells Pootle FS how to synchronize the translations
+in the project. The default option is ``localfs`` which tells to synchronize
+with the disk. Other backends allow to synchronize with version control systems
+and they might require the installation of additional Pootle FS plugins.
+
+**Path or URL** either is a path pointing to the translation files on the disk
+if you are using the ``localfs`` **Filesystem backend**, or is a URL pointing
+to the VCS repository. The path can be an absolute path like
+``/path/to/translations/my-project/`` or it can use the
+``{POOTLE_TRANSLATION_DIRECTORY}`` placeholder if you are using the ``localfs``
+**Filesystem backend**, like ``{POOTLE_TRANSLATION_DIRECTORY}my-project``
+(the placeholder will be transparently replaced by the value of the
+:setting:`POOTLE_TRANSLATION_DIRECTORY` setting).
```

Comment:
I'd add a similar verbose instance of a proper non-localfs backend e.g. git.  Though you do that later which might be enough, just worried this leaves people with a localfs impression, rather than a fuller picture.